packages/mui-base/src/utils/useSlotProps.ts

```diff
@@ -72,8 +76,16 @@ export default function useSlotProps<
     OwnerState
   >,
 ) {
-  const { elementType, externalSlotProps, ownerState, ...rest } = parameters;
-  const resolvedComponentsProps = resolveComponentProps(externalSlotProps, ownerState);
+  const {
+    elementType,
+    externalSlotProps,
+    ownerState,
+    skipResolvingSlotProps = false,
```

Comment:
Could you add a unit test for it?

Reply:
Right, forgot about this, I only added for the Slider. Adding it right now 👍

Reply:
Done.